src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
`enum Color`

Reply:
Could you please clarify your comment? Is it considered bad practice to use plural to name entities in Java?